app/src/main/java/org/wikipedia/games/onthisday/OnThisDayGameNotificationManager.kt

```diff
@@ -0,0 +1,143 @@
+                Prefs.otdNotificationState = OnThisDayGameNotificationState.ENABLED
+                activity.invalidateOptionsMenu()
+            }
+        )
+    }
+
+    private fun showEnabledNotificationDialog() {
+        OnThisDayGameDialogs.showTurnOnNotificationDialog(
+            activity = activity,
+            turnThemOnButtonOnClick = {
+                enableNotifications(showUndo = true)
+            },
+            keepThemOffButtonOnclick = {
+                Prefs.otdNotificationState = OnThisDayGameNotificationState.DISABLED
+                activity.invalidateOptionsMenu()
+            }
+        )
+    }
+
+    private fun disableNotifications(showUndo: Boolean) {
+        Prefs.otdNotificationState = OnThisDayGameNotificationState.DISABLED
+        cancelDailyGameNotification(activity)
+        if (showUndo) {
+            FeedbackUtil.makeSnackbar(
+                activity,
+                activity.getString(R.string.on_this_day_game_notification_turned_off_snackbar_message)
+            ).apply {
+                setAction(R.string.reading_list_item_delete_undo) {
+                    enableNotifications(showUndo = false)
+                    activity.invalidateOptionsMenu()
+                }
+            }.show()
+        }
+        activity.invalidateOptionsMenu()
+    }
+
+    private fun enableNotifications(showUndo: Boolean) {
+        Prefs.otdNotificationState = OnThisDayGameNotificationState.ENABLED
+        (activity as? OnThisDayGameActivity)?.requestPermissionAndScheduleGameNotification()
+        if (showUndo) {
+            FeedbackUtil.makeSnackbar(
+                activity,
+                activity.getString(R.string.on_this_day_game_notification_turned_on_snackbar_message)
+            ).apply {
+                setAction(R.string.reading_list_item_delete_undo) {
+                    disableNotifications(showUndo = false)
+                    activity.invalidateOptionsMenu()
+                }
+            }.show()
+        }
+        activity.invalidateOptionsMenu()
+    }
+
+    companion object {
+        private const val NOTIFICATION_TYPE_LOCAL = "local"
+
+        fun showNotification(context: Context) {
+            if ((WikipediaApp.instance.getResumedActivity() is OnThisDayGameActivity).not()) {
+                NotificationPresenter.showNotification(
+                    context = context,
+                    builder = NotificationPresenter.getDefaultBuilder(context, 1, NOTIFICATION_TYPE_LOCAL),
+                    id = 1,
+                    title = context.getString(R.string.on_this_day_game_feed_entry_card_heading),
+                    text = context.getString(R.string.on_this_day_game_notification_text),
+                    longText = context.getString(R.string.on_this_day_game_notification_text),
+                    lang = null,
+                    icon = null,
+                    color = R.color.blue600,
+                    bodyIntent = OnThisDayGameActivity.newIntent(
```

Comment:
Edge case: should we detect the "close" can go back to the Explore feed instead of exiting the app?

Steps to reproduce:
1. Show the notification by tapping on the dev preference item.
2. Close the app entirely.
3. Click on the notification to play the game.
4. Click on the close button.